### ISSUE TITLE
Instalation help

### ISSUE THREAD

**Issue by Albert1st1** [Syniurge/i2c-amd-mp2#9]
git clone https://github.com/Syniurge/i2c-amd-mp2.git master
cd master
LANG=C ./dkms-install.sh 

About to run dkms install steps...
cp: cannot stat '../i2c-amd-mp2': No such file or directory
Error! Could not find module source directory.
Directory: /usr/src/i2c-amd-mp2-1.0.0 does not exist.
Error! Could not find module source directory.
Directory: /usr/src/i2c-amd-mp2-1.0.0 does not exist.
Error! Could not find module source directory.
Directory: /usr/src/i2c-amd-mp2-1.0.0 does not exist.
Finished running dkms install steps.

The install script doesn't look for files in current working directory but rather ../i2c-amd-mp2

What did i do wrong?
I use LANG=C so outputs are in english instead of my native language.

**Comment by tobiasThomer** [comment]
Hey, 
line 10 says
cp -r ../${DRV_DIR} /usr/src/${DRV_NAME}-${DRV_VERSION}
and line 8 
DRV_DIR=i2c-amd-mp2

try to rename the folder from "master" to "i2c-amd-mp2" ;)

**Comment by Albert1st1** [comment]
I did try that and then i needed to move more files around.
Then in the end it got compile error complaining about missing Makefile.
I think the install should work as described in the README.md

**Comment by tobiasThomer** [comment]
Strange, you should not have to move files. Can you post the error messages? Do you have the kernel headers installed?

**Comment by Albert1st1** [comment]
I have both headers and source.
I did removed my other kernels just to be sure they was not the cause. 
I removed everything i downloaded and started over.

git clone https://github.com/Syniurge/i2c-amd-mp2.git master
mv master i2c-amd-mp2
cd i2c-amd-mp2
LANG=C ./dkms-install.sh

Then the Install worked as it should.
To bad touchpad still doesn't work.

Edit: I tried different boot options and now my touchpad is working :smiley: 
I had pci=noacpi and the touchpad didn't work.
Changed to:
ivrs_ioapic[4]=00:14.0 ivrs_ioapic[5]=00:00.2
Now touchpad works.
It's an Acer A315-41-R6VQ
